### ISSUE TITLE
envsubst

### ISSUE THREAD

**Issue by aaronstuder** [Ubiquiti-App/UNMS#76]
This script requires 'envsubst'. Please install 'envsubst' and try again. Aborting.

Tried both Ubuntu 16.04.1 LTS (Xenial Xerus) 64-bit and Debian 8 64-bit on Scaleways.

**Comment by jaroklima** [comment]
@aaronstuder I don't think the error message can be much clearer. The script requires [envsubst](http://manpages.ubuntu.com/manpages/xenial/man1/envsubst.1.html). If it's not available on your system, please install it using the standard package manager
`sudo apt-get install envsubst`

**Comment by nickwinn** [comment]
You went thru all that trouble @jaroklima and you didn't bother mentioning this is provided by gettext-base package. 

envsubst is not a package name. 

ಠ_ಠ

**Comment by jaroklima** [comment]
You're right, thank you. I will update the error message with the package name. Sorry.

**Comment by ChrysUk** [comment]
Any news regarding this ? The error is still there

**Comment by radekskrivan** [comment]
Hello @ChrysUk. Can you please be more specific? The error message has been updated to 
`This script requires 'envsubst'. Please install 'gettext-base' and try again` so I am not sure what error you are referring to.
